target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dts

```diff
@@ -381,21 +381,73 @@
 	pinctrl-0 = <&mdio0_pins>;
 	pinctrl-names = "default";
 
-	phy0: ethernet-phy@0 {
-		reg = <0>;
-		qca,ar8327-initvals = <
-			0x04 0x80080080  /* PAD0_MODE */
-			0x0c 0x06000000  /* PAD6_MODE */
-			0x10 0x002613a0  /* PWS_REG */
-			0x50 0xcc36cc36  /* LED_CTRL0 */
-			0x54 0xca36ca36  /* LED_CTRL1 */
-			0x58 0xc936c936  /* LED_CTRL2 */
-			0x5c 0x03ffff00  /* LED_CTRL3 */
-			0x7c 0x0000004e  /* PORT0_STATUS */
-			0x94 0x0000004e  /* PORT6_STATUS */
-			0xe0 0xc74164de  /* SGMII_CTRL */
-			0xe4 0x0006a545  /* MAC_PWR_SEL */
-			>;
+	switch@10 {
+		compatible = "qca,qca8337";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x10>;
+
+		qca8k,mac6_exchange;
+		qca8k,rgmii56_1_8v;
+		qca8k,led_blink_freq = <8>;
```

Comment:
It seems that `/bits/ 8` is needed to use `of_property_read_u8`.

Reply:
I used u8 since the value is very small... was thinking of just dropping this and use u32